knowage-vue/src/modules/kpi/kpiDocumentDesigner/KpiDocumentDesigner.vue

```diff
@@ -45,13 +45,18 @@ import KpiDocumentDesignerSaveDialog from './KpiDocumentDesignerSaveDialog/KpiDo
 import KpiDocumentDesignerScorecardsListCard from './KpiDocumentDesignerScorecardsListCard/KpiDocumentDesignerScorecardsListCard.vue'
 import { mapState } from 'pinia'
 import mainStore from '../../../App.store'
+import UserFunctionalitiesConstants from '@/UserFunctionalitiesConstants.json'
 
 import deepcopy from 'deepcopy'
 
 export default defineComponent({
     name: 'kpi-document-designer',
     components: { KpiDocumentDesignerDocumentTypeCard, KpiDocumentDesignerKpiListCard, KpiDocumentDesignerOptionsCard, KpiDocumentDesignerStyleCard, KpiDocumentDesignerTypeCard, KpiDocumentDesignerSaveDialog, KpiDocumentDesignerScorecardsListCard },
     props: { id: { type: String } },
+    setup() {
```

Comment:
You can remove the setup because the "user" object is within the map state you have already imported.

Reply:
Fixed